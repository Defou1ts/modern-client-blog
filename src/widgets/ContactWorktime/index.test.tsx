import React from 'react';

import { render, screen } from '@testing-library/react';

import renderer from 'react-test-renderer';

import { ContactWorkTime } from './index';

describe('ContactWorkTime', () => {
	it('renders a  ContactWorkTime', () => {
		render(<ContactWorkTime />);
		const text = screen.getByText('contact.working.title-days');
		expect(text).toBeInTheDocument();
		expect(text).toBeVisible();
	});

	it('ContactWorkTime snapshot', () => {
		const component = renderer.create(<ContactWorkTime />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
