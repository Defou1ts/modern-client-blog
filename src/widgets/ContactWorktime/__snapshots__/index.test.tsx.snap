// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactWorkTime ContactWorkTime snapshot 1`] = `
<section
  className="wrapper"
>
  <div>
    <p
      className="p small itemTitle"
    >
      contact.working.title-days
    </p>
    <div
      className="line"
    />
    <h5
      className="h h5 info"
    >
      contact.working.days
       
      <br />
       
      contact.working.hours
       
    </h5>
    <p
      className="p medium support"
    >
      contact.working.availability
    </p>
  </div>
  <div>
    <p
      className="p small itemTitle"
    >
      contact.working.title-contact
    </p>
    <div
      className="line"
    />
    <h5
      className="h h5 info"
    >
      020 7993 2905
    </h5>
    <p
      className="p medium support"
    >
      hello@finsweet.com
    </p>
  </div>
</section>
`;
